sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## 5.1.0b8 (Unreleased)
 
+**Breaking Changes**
+- Changed the response structure of `being_analyze_actions`. Now, we return a list of list of an action result of documents
```

Comment:
I think this should be "list of list**s**"? Also wondering if we can clarify the change a bit more - maybe "Now, we return a list of results, where each result is a list of the action results for the document, in the order the actions were passed". If that's more confusing then ignore me

Reply:
I think i like that more, thank you @kristapratico !